src/static/css/page.css

```diff
@@ -277,14 +277,23 @@
 
 .authors .social {
   display: inline-block;
-  vertical-align: middle;
+  vertical-align: top;
+  font-size: 0.9em;
 }
 
 .authors .social a {
-  display: inline-block;
-  vertical-align: middle;
-  margin: 0 5px;
-  margin: 0 0.3125rem;
+  margin: 0 10px 0 0;
+  margin: 0 0.625rem 0 0;
+  text-decoration: none;
+  white-space: nowrap;
+}
+
+.authors .social svg {
+  width: 1em;
+  height: auto;
+  vertical-align: -0.125rem;
```

Comment:
This seems hacky. Why doesn't `middle` work?

Reply:
Yeah this is only needed if putting it on the same line as Author name as middle seems
to break then. Suspect because of two different font sizes between name and author. As you’re happy to always have this on second line then can remove this hack.

Reply:
Removed.